CMakeLists.txt

```diff
@@ -150,6 +159,25 @@ if (${TRITON_PYTORCH_DOCKER_BUILD})
 
   string(REPLACE ";" " " CONDA_LIBS_STR "${CONDA_LIBS}")
 
+  if (${TRITON_PYTORCH_ENABLE_TORCHTRT})
+    execute_process(
+      COMMAND /bin/sh -c "rm -fr /tmp/ttrt.so; docker rm -f pytorch_backend_ptlib > /dev/null 2>&1 || true; docker create --name pytorch_backend_ptlib ${TRITON_PYTORCH_DOCKER_IMAGE} && docker cp pytorch_backend_ptlib:/opt/conda/lib/python3.8/site-packages/torch_tensorrt/lib/libtorchtrt_runtime.so /tmp/ttrt.so && docker rm pytorch_backend_ptlib"
+      RESULT_VARIABLE TORCHTRT_STATUS
+      OUTPUT_VARIABLE TORCHTRT_OUT
+      ERROR_VARIABLE  TORCHTRT_ERR
+    )
+    # message("TORCHTRT_STATUS is ${TORCHTRT_STATUS}\nTORCHTRT_OUT is ${TORCHTRT_OUT}\nTORCHTRT_ERR is ${TORCHTRT_ERR}")
```

Comment:
cleanup

Reply:
@CoderHam : yes that's exactly what I did for server - that does not let the user override the option though, and would fail if an older Pytorch image is used (I do see pipelines running now for 21.08 etc).